moped-editor/src/utils/activityLogFormatters/mopedContractsActivity.js

```diff
@@ -58,10 +59,20 @@ export const formatContractsActivity = (change) => {
   const oldRecord = change.record_data.event.data.old;
 
   let changes = [];
+  const fieldsToSkip = ["updated_at", "updated_by_user_id"];
 
   // loop through fields to check for differences, push label onto changes Array
   Object.keys(newRecord).forEach((field) => {
-    if (newRecord[field] !== oldRecord[field]) {
+    // typeof(null) resolves as "object", check that field is not null before checking if object
+    // task orders are in arrays
+    if (!!newRecord[field] && typeof newRecord[field] === "object") {
```

Comment:
i don't think there's any harm in checking all the fields, but you could compare the task orders' `task_order` property, which is the unique task order ID.